recruitr/settings.py

```diff
@@ -123,7 +135,7 @@
 
 LANGUAGE_CODE = 'en-us'
 
-TIME_ZONE = 'America/Los_Angeles'
+TIME_ZONE = 'Asia/Kolkata'
```

Comment:
can you replace the timezones with "Etc/UTC" so that we don't give any default preference?